yarn.lock

```diff
@@ -1686,6 +1686,13 @@
   resolved "https://registry.yarnpkg.com/@fluentui/react-icons/-/react-icons-2.0.159-beta.10.tgz#8934c05b415bbb1b46f9085c72eaf78fd7bad967"
   integrity sha512-Y2pDtl6ofeqGI/M/C3T6KbFy+deT1TkQ5aMO3TWN4XgAYQb5Sww56WeuX+sQNnnIXN21HIJOS6Hw1frFtCPv0w==
 
+"@fluentui/react-theme@9.0.0-rc.1":
```

Comment:
curious why was this added ? react-theme latest is `9.0.0-rc.4` so I'd suggest we use that one, thus removing this entry to prevent dupes.

Reply:
@Hotell ideally I don't want to take the dependency on "react-theme", we'd likely prefer to just have a set of tokens, with defaults. This will go away, but we needed to ensure we had interop w/ the React Theme Provider in the demo, and this is where the tokens currently come from.

I'll look to update and drop this dependency.

Reply:
sounds good, we should spin a new package just for token that will be platform/rendering layer agnostic